perf/src/sigverify.rs

```diff
@@ -486,94 +477,6 @@ pub fn generate_offsets(
     )
 }
 
-pub struct Deduper {
```

Comment:
It looks like only Solana crates depend on this directly according to https://crates.io/crates/solana-perf/reverse_dependencies, so it's safe to move without a `pub use`

Reply:
I think we still need/want `pub` for two reasons:
- `Deduper` isn't instantiated in non-test/bench code in `solana-perf`, so `cargo` complains about dead code when building `solana-perf`.
- More importantly, `SigverifyStage` is the consumer of `Deduper`, `SigverifyStage` comes from `solana-core` so we need visibility beyond `solana-perf` (ie `pub(crate)` wouldn't work here either)

Reply:
oh sorry, I meant putting in a `pub use crate::deduper::Deduper` in this file so that outside users can still do `use solana_perf::sigverify::Deduper`. Agreed that this definitely needs to stay `pub`

Reply:
Oh d'oh; I misinterpreted but see what you mean now.

In any case, thanks for double checking. I admittedly assumed this was an internal thing, but while checking crate registry isn't foolproof, better than assuming